src/_locales/en/messages.json

```diff
@@ -9,7 +9,7 @@
     "description": "Name of the extension."
   },
   "extensionDescription": {
-    "message": "Allows you to do great stuff.",
+    "message": "Allows you to autocorrect common text sequences and convert text characters to a look like a special Unicode font.",
```

Comment:
Since the end will get cut off in the Add-on manager, maybe shorten it to something like this:
>Autocorrect common symbols and transform text to Unicode fonts.

Reply:
It may be abbreviated in the overview, but even the German version is displayed completely when you open the settings:
![grafik](https://user-images.githubusercontent.com/11966684/119990535-bbcbeb80-bfc8-11eb-80e7-98ead548a855.png)

So I personally see nothing wrong with having it a little longer…
As long as you can notice what the add-on was about when scrolling through the add-ons page. :upside_down_face:

Reply:
If is OK with you, it is fine with me… When scrolling through my installed add-ons, I just personally thought that ones with shorter and more concise descriptions looked better:
![image](https://user-images.githubusercontent.com/15305150/119994935-f1011b00-bf81-11eb-822a-87ea10cce703.png)

Anyway, for style maybe we should not use the word “text” twice. Also, the “Allows you to” is implied, so should not be needed.